lib/parser/characters.cc

```diff
@@ -122,8 +122,9 @@ template<> EncodedCharacter EncodeCharacter<Encoding::UTF_8>(char32_t ucs) {
 EncodedCharacter EncodeCharacter(Encoding encoding, char32_t ucs) {
   switch (encoding) {
   case Encoding::LATIN_1: return EncodeCharacter<Encoding::LATIN_1>(ucs);
-  case Encoding::UTF_8: return EncodeCharacter<Encoding::UTF_8>(ucs);
-  default: CRASH_NO_CASE;
+  case Encoding::UTF_8:
+    return EncodeCharacter<Encoding::UTF_8>(ucs);
+    SWITCH_COVERS_ALL_CASES
```

Comment:
clang-format treates `SWITCH_COVERS_ALL_CASES` like it is part of the last case.
What do you think of putting it between the `switch` and the first `case` to make it clear it is separate?
```
  switch (encoding) {
    SWITCH_COVERS_ALL_CASES
  case Encoding::LATIN_1: return EncodeCharacter<Encoding::LATIN_1>(ucs);
  case Encoding::UTF_8: return EncodeCharacter<Encoding::UTF_8>(ucs);
  }
```